### ISSUE TITLE
dark-percentage not working in QBtn API

### ISSUE THREAD

**Issue by timw8890** [quasarframework/quasar#13333]
### What happened?

The dark-percentage attribute doesn't seem to have any effect.

### What did you expect to happen?

That the percentage is shown in a darker color.

### Reproduction URL

https://jsfiddle.net/ya0u61o5/1/

### How to reproduce?

Go to provided reproduction URL and click the button "Compute PI". Notice how the percentage is not visible on a white button.

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19044) - win32/x64
NodeJs - 14.17.3

Global packages
  NPM - 8.3.2
  yarn - Not installed
  @quasar/cli - 1.2.1
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.6.6 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 3.3.3 -- Quasar Framework local CLI
  @quasar/extras - 1.13.6 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.29 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.12
  vuex - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.15.8 -- Babel compiler core.
  webpack - 5.58.2 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. 
json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.7.4 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - DESKTOP-74J4ODK
  vEthernet (WSL) - 172.26.32.1
  VirtualBox Host-Only Network - 192.168.56.1
  VMware Network Adapter VMnet1 - 192.168.65.1
  VMware Network Adapter VMnet8 - 192.168.40.1
  WLAN Antenne - 192.168.0.115
```

**Comment by pdanpdan** [comment]
That prop only changes the background overlay (by default it is white transparent - when using dark-percentage it is  black transparent) - you still have to use a color for your button text to make it visible

**Comment by timw8890** [comment]
@pdanpdan 
> That prop only changes the background overlay

I guess you mean the progress bar that appears after the button has been clicked. When using dark-percentage on a white button this black-transparent overlay should be visible then, right? In my code example it is not visible. But maybe I have misunderstood something.

**Comment by pdanpdan** [comment]
You're right, wrong CSS selector for progress dark :)

**Comment by pdanpdan** [comment]
Fix will be available in v2.6.7/V1.18.11
